__tests__/add_rem.test.js

```diff
@@ -0,0 +1,54 @@
+describe('tests', () => {
```

Comment:
- [ ] Other _pure functions_ in your code are **not tested** aside _add item_ and _delete item_. Kindly ensure to test all pure functions such as the _edit_, _complete: true/false_, and also _clear_ all completed_

Reply:
Thank you for your review.. Is there anything else needed?

Reply:
@Somdotta07 
The above should be good!

Cheers!